### ISSUE TITLE
Best practices not required in WCAG

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-quick-start#103]
In Easy Checks (and I think the Tutorials) we were very careful if we put things in there that were not *required* by WCAG, that we stated it was a best practice, but not required. I wonder if we need to do that in these Tips as well?

For example: in Tips for Designing, Ensure interactive elements are easy to identify, touch-screen activation: http://w3c.github.io/wai-quick-start/designing.html#ensure-interactive-elements-are-easy-to-identify

(e-mail thread: https://lists.w3.org/Archives/Public/wai-eo-editors/2015Jul/0025.html)

**Comment by nitedog** [comment]
If the *tip* is not a requirement, then I strongly advocate to remove it completely - this is a start-up guide to get people going - best practices come later on.

However, if the example stretches a bit, like the activation style which is borderline in WCAG, then I don't think it is as big of an issue because it is just providing a complete example.

**Comment by yatil** [comment]
_Technically_ responsive design is not required by WCAG either, yet I feel strongly to keep it in. This is not a normative resource and is not covering WCAG in its entirety, so having some sidenotes that further improve does not hurt. In the RWD example it uses learned design practice and shows how it applies to accessibility. That is not a baseline conformance but helps to convey the message.

**Comment by iadawn** [comment]
Based on discussion between Shawn and Kevin, the two issues noted (focus and responsive design) will be labelled as Best Practice (somehow). Need to keep this in mind for other techniques.

**Comment by iadawn** [comment]
I have included a couple of approaches to indicate best practice (7bc93e9). The tip on focus I find a little bit difficult to write for as it is only really if you do any styling should you provide alternative styling, and keyboard focus is the only requirement, the rest are best practice.

**Comment by iadawn** [comment]
Items that could possibly be identified as best practice rather than clear requirements:

* Designing
  * [Ensure that interactive elements are easy to identify](http://w3c.github.io/wai-quick-start/designing.html#ensure-that-interactive-elements-are-easy-to-identify) - touch activation, and hover
  * [Provide easily identifiable feedback](http://w3c.github.io/wai-quick-start/designing.html#provide-easily-identifiable-feedback) - present feedback in a prominent style
  * [Use spacing and headings to group related content](http://w3c.github.io/wai-quick-start/designing.html#use-spacing-and-headings-to-group-related-content) - SC relates to providing descriptive headings not how they are presented or used to group content
  * [Create designs for different viewport sizes](http://w3c.github.io/wai-quick-start/designing.html#create-designs-for-different-viewport-sizes) - Responsive design isn't covered by SC
* Writing
  * [Provide informative, unique page titles](http://w3c.github.io/wai-quick-start/writing.html#provide-informative-unique-page-titles) - Arguably, providing the stage within a process is best practice
  * [Keep content clear and concise](http://w3c.github.io/wai-quick-start/writing.html#keep-content-clear-and-concise) - Is the use of lists as appropriate an SC?
* Developing
  * [Write code that adapts to user's technology](http://w3c.github.io/wai-quick-start/developing.html#write-code-that-adapts-to-users-technology)
  * [Avoid CAPTCHA where possible](http://w3c.github.io/wai-quick-start/developing.html#avoid-captcha-where-possible)

**Comment by James-Green** [comment]
Adding best practice labels is fine with me, but I don't think we should remove items we agree are valuable because they are not a 1:1 match to WCAG.  I *think* our intent is to provide tips that help improve web accessibility using WCAG 2.0 as our starting point and standard, not just tips to meet the letter of the top WCAG 2.0 requriements.  Some tips are so specific they can be mapped 1:1 to WCAG, but others are more broad. I base part of my opinion on the message we send in the first sentence of Writing Tips as it stands today: 

"This page helps you get started writing web content that is more accessible to people with disabilities. These tips introduce some basic accessibility considerations. Each tip includes links to related WCAG requirements and resources to help you understand and implement the tip."

**Comment by nitedog** [comment]
I think the comparison to EasyChecks and Tutorials could be misleading - these resources tell you how to check or do something, while the tips are just - well - tips. It is not only the 1:1 mapping to Success Criteria that is missing, but also the actual wording includes "good practice" rather than "must do to meet WCAG". I think it is better to work on the framing in the intro paragraphs of the overview page and individual sub-pages, to make sure that people understand that these tips only lead you in the right direction, but that they are not the actual requirements.

**Comment by iadawn** [comment]
Addressed with changes to introcution 61fde42

Related to [EO survey of 11 Sep](https://www.w3.org/2002/09/wbs/35532/EOWG-11Sep2015/results#xdevelTp)

**Comment by yatil** [comment]
👍
